### ISSUE TITLE
Performance

### ISSUE THREAD

**Issue by erdbeere** [erdbeere/fliesen-wiederspiel#3]
Geht es auch ohne dass der Lüfter hochdreht?

**Comment by admonstrator** [comment]
Nein. Allein schon, weil hunderttausendmal favicon.svg geladen wird.
Das ist doch ein versteckter DDOS.

**Comment by erdbeere** [comment]
Dass das Favicon mehrfach heruntergeladen wird konnte ich so noch nicht beobachten. Kannst du mir dabei helfen, das Issue zu reproduzieren, @Admonstrator?
